spec/mail/encodings_spec.rb

```diff
@@ -837,7 +838,7 @@
   describe ".charset_encoder" do
     class CustomEncoder
       def encode(str, charset)
-        "#{str}-#{charset}"
+        String.new("#{str}-#{charset}")
```

Comment:
yes - unless Ruby19.q_value_decode is changed to not call force_encoding on the result of it